README.md

```diff
@@ -131,31 +131,38 @@ Depending on your preference, you can also add parameters as named arguments, in
 response = await dg_client.transcription.prerecorded(source, punctuate=True, keywords=['first:5', 'second'])
 ```
 
-## Code Samples
-
-To run the sample code, you may want to create a virtual environment to isolate your Python projects, but it's not required. You can learn how to make and activate these virtual environments in [this article](https://blog.deepgram.com/python-virtual-environments/) on our Deepgram blog.
-
-#### Streaming Audio Code Samples
+## Testing
 
-In the `sample-projects` folder, there are examples from four different Python web frameworks of how to do live streaming audio transcription with Deepgram. These include:
+### Setup
 
-- Flask 2.0
-- FastAPI
-- Django
-- Quart
+```
+pip install pytest
```

Comment:
We have a [requirements-dev.txt](https://github.com/deepgram/deepgram-python-sdk/blob/main/requirements-dev.txt) file in the repo that includes `pytest` and some other dependencies, so we can tell the user to run `pip install -r requirements-dev.txt` instead.

`pytest-cov` needs to be added to that file.

Reply:
Should I remove the instructions to `pip install pytest-cov` and `pip install pytest` since these are in the requirements-dev.txt file?

Reply:
Yeah, you could replace those two `pip install` lines with `pip install -r requirements-dev.txt`